build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg

```diff
@@ -188,18 +189,17 @@ fi
 %post
 ostype="#LINUXDIST#"
 if [ $ostype == "CentOS" ]; then
-  sed -i '/enabled=/d' /etc/yum.repos.d/CentOS-Base.repo;
-  sed -i '/\[.*\]/ a\
-enabled=0' /etc/yum.repos.d/CentOS-Base.repo;
+  sed -i -e '/enabled=/d' -e '/^\[.*\]/ aenabled=0' /etc/yum.repos.d/CentOS-Base.repo;
```

Comment:
`aenabled`

Reply:
This was sed magic and it should be fine, but maybe a comment explaining why `aenabled=0` isn't a typo would be great

Reply:
It _also_ works with `-e '/^\[.*\]/a enabled=0'`, so I've changed it to that. 👍